### PR TITLE
Update/search improvements

### PR DESCRIPTION
**Pull request by simison**
- Small fix to user search (Got `.finally is not a function` when searching)
- Link to search for various places in the UI

Kinda supersedes #1020

### CODE REVIEW

File: modules/search/client/components/SearchUsers.component.js [hunk 1]
```diff
@@ -67,18 +67,16 @@ class SearchUsers extends React.Component {
...
+  actionSearch(event) {
     event.preventDefault();
     event.stopPropagation();
 
     this.setState({ isSearching: true }, () => {
-      searchUsers(this.state.searchQuery)
-        .then(({ data }) => {
-          this.setState({ users: data });
-        })
-        .finally(() => {
```
Comment (simison) [inline modules/search/client/components/SearchUsers.component.js]: @mrkvon do you know why `.finally` didn't work here?

@Akronix and I were expecting `searchUsers()` to return a standard promise and thus have `finally`. 🤔

Comment (mrkvon) [reply modules/search/client/components/SearchUsers.component.js]: One possibility - you have a browser which [supports `Promise`](https://caniuse.com/#feat=promises), but [not `Promise.prototype.finally`](https://caniuse.com/#feat=promise-finally). That would also mean that we don't [transpile Promises](https://stackoverflow.com/questions/38960490/how-can-i-polyfill-promise-with-webpack) and we should.

**Comment by simison** [comment]
Latest Firefox, should support ".finally" I suppose.

File: modules/search/client/components/SearchUsers.component.js (hunk 1 above)
Comment (mrkvon) [reply modules/search/client/components/SearchUsers.component.js]: > Latest Firefox, should support ".finally" I suppose.

I can confirm the error before the fix.

Comment (Akronix) [reply modules/search/client/components/SearchUsers.component.js]: I think this is not critical. We can leave it as it is for now. My only concern is that the search members feature works and this isn't there yet. So I'd prioritize to have the indexing working.

Yep, the async was wrongly added to actionSearch, but we already removed it in this PR.

**Pull request merged**